src/contrib/mlir/backend/pass/affine_lowerer.cpp

```diff
@@ -1410,6 +1476,30 @@ namespace
         }
         NGRAPH_UNREACHABLE("Unsupported type");
     }
+
+    ValueHandle createOneConstant(mlir::Type type)
```

Comment:
Can you unify this and the func above to `template<typename T> createConstant(mlir::Type type, T value)`